dev/tools/make-changelog.sh

```diff
@@ -35,7 +35,7 @@ if [ ! -z "$fixes_string" ]; then fixes_string="$(printf '\n  fixes %s,' "$fixes
 # the ` are regular strings, this is intended
 # use %s for the leading - to avoid looking like an option (not sure
 # if necessary but doesn't hurt)
-printf '%s **%s:**\n  Bla bla\n  (`#%s <https://github.com/coq/coq/pull/%s>`_,%s\n  by %s).' - "$type_full" "$PR" "$PR" "$fixes_string" "$(git config user.name)" > "$where"
+printf '%s **%s:**\n  Describe your change here but don't end with a period\n  (`#%s <https://github.com/coq/coq/pull/%s>`_,%s\n  by %s)' - "$type_full" "$PR" "$PR" "$fixes_string" "$(git config user.name)" > "$where"
```

Comment:
Hi, this happens to trigger a syntax error because of the `'` in `don't` (at least on MacOS). A possible fix which works for me is to write `don'"'"'t`.

Reply:
We could also change it to "do not"

Reply:
Done

Reply:
Wow, that's quick. Wonderful.